optuna/visualization/matplotlib/_optimization_history.py

```diff
@@ -51,11 +56,16 @@ def objective(trial):
             study = optuna.create_study(sampler=sampler)
             study.optimize(objective, n_trials=10)
 
+            plt.tight_layout()
             optuna.visualization.matplotlib.plot_optimization_history(study)
 
+        .. note::
+            You need  to adjust the size of the plot by yourself using `plt.tight_layout()` or `plt.savefig(IMAGE_NAME, bbox_inches='tight')`.
```

Comment:
thanks! how about this?
```suggestion
            You need to adjust the size of the plot by yourself using ``plt.tight_layout()`` or
            ``plt.savefig(IMAGE_NAME, bbox_inches='tight')``.
```